protocol/messenger.go

```diff
@@ -3112,12 +3112,28 @@ func (m *Messenger) MarkAllRead(chatID string) error {
 // MuteChat signals to the messenger that we don't want to be notified
 // on new messages from this chat
 func (m *Messenger) MuteChat(chatID string) error {
+	// Only one to one chan be muted
+	publicKey, err := common.HexToPubkey(chatID)
+	if err != nil {
+		return err
+	}
+
 	chat, ok := m.allChats.Load(chatID)
 	if !ok {
```

Comment:
Wondering why chat absence is not an error during muting?

Reply:
No, it's the case where you for example see someone in a public chat, and click on profile, and decide to mute them, without opening a chat with them.
In this case, the chat is not created, but we need to create on and mute it.

Reply:
Got it, thanks!